modules/node-tuning-operator-supported-tuned-daemon-plug-ins.adoc

```diff
@@ -24,15 +24,20 @@ using custom profiles defined in the `profile:` section of the Tuned CR:
 * usb
 * video
 * vm
+* bootloader
 
 There is some dynamic tuning functionality provided by some of these plug-ins
 that is not supported. The following TuneD plug-ins are currently not supported:
 
-* bootloader
 * script
 * systemd
 
 
+[WARNING]
+====
+The TuneD `[bootloader]` plug-in is currently supported on {op-system-first} 8.x worker nodes. For {op-system-base-full} 7.x worker nodes, the TuneD `[bootloader]` plug-in is currently not supported.
```

Comment:
Is there a specific reason to specify `[bootloader]` is square brackets in this note, but not in the bulleted list? If there is not, you could consider using a consistent format throughout.

Also, the `[bootloader]` plug-in reference in this note is in back-ticks, but the references to each of the plug-ins in the bulleted list are not in back-ticks. Are these plug-in names referencing code elements? You could update the references in the bulleted list and the node to either consistently use back-ticks or not use back-ticks.

This also applies to the note in `modules/configuring-huge-pages.adoc` that is also updated in this PR.

Reply:
Addressed